packages/ibm-products/src/components/DataSpreadsheet/DataSpreadsheet.tsx

```diff
@@ -291,7 +291,11 @@ export let DataSpreadsheet = React.forwardRef(
                   activeCellCoordinates?.column
                 ]
               : null;
-          setActiveCellContent(activeCellFullData.render('Cell'));
+          if (activeCellFullData) {
+            setActiveCellContent(activeCellFullData.render('Cell'));
+          } else {
+            setActiveCellContent(null);
+          }
```

Comment:
What is odd is not sure why we allow `activeCellFullData` to be `null` in this condition block here but then try to do a `.render('cell')` on it. Seems a use case was missed here. So my fix should patch that, thanks!

![Screenshot 2024-05-20 at 1 39 49 PM](https://github.com/carbon-design-system/ibm-products/assets/10100397/5a60c2d2-094c-4785-893e-ad4afb1cbfc3)